app/Resources/translations/default/ShopThemeCheckout.xlf

```diff
@@ -308,12 +311,17 @@ File: themes/StarterTheme/templates/checkout/order-confirmation.tpl:62</note>
       <trans-unit id="856ab109229e8acfcddd5c6766a8d956">
         <source>Subtotal:</source>
         <target>Subtotal:</target>
-        <note>Line: 59</note>
+        <note>Line: 74</note>
       </trans-unit>
       <trans-unit id="66f51e43606e2baebb8e9c3bfe965da4">
         <source>Shipping:</source>
         <target>Shipping:</target>
-        <note>Line: 61</note>
+        <note>Line: 76</note>
+      </trans-unit>
+      <trans-unit id="b208de3d9294511d4001606e63775185">
+        <source>There is %products_count% item in your cart.</source>
```

Comment:
Do we have a plural option for this string?

Reply:
In this case two different wordings have been used:
```smarty
              {if $cart.products_count > 1}
                <p class="cart-products-count">{l s='There are %products_count% items in your cart.' sprintf=['%products_count%' => $cart.products_count] d='Shop.Theme.Checkout'}</p>
              {else}
                <p class="cart-products-count">{l s='There is %products_count% item in your cart.' sprintf=['%products_count%' =>$cart.products_count] d='Shop.Theme.Checkout'}</p>
              {/if}
```